Makefile

```diff
@@ -56,4 +56,4 @@ dev:
 	$(DEPS_BIN_DIR)/mocha test/*-test.js $(MOCHA_FLAGS) --watch
 
 watch:
-	$(DEPS_BIN_DIR)/babel -w src --out-dir lib
+	$(DEPS_BIN_DIR)/babel -w src --out-dir lib -s inline
```

Comment:
Can you prefix the script with `NODE_ENV=dev` or something similar and then only `require` in `'source-map-support/register'` if `NODE_ENV === 'dev'`.